dump_xtc installation error

Hi everbody,
I tried to install the .xtc extension in a Docker Container (Ubuntu 17.10 and 18.04) with GROMACS not installed in standard directory (i.d. not using apt-get). Every time running the installation with

cmake -DWITH_XTC=ON -DGROMACS_LIBRARIES=${gromacs_prefix}/lib/-DGROMACS_INCLUDES=${gromacs_prefix}/include .

I received following error message:

[ 93%] Building CXX object src/CMakeFiles/_espressopp.dir/io/bindings.cpp.o
In file included from /Downloads/espressopp-1.9.5/src/io/DumpXTC.cpp:23:0:
/Downloads/espressopp-1.9.5/src/io/DumpXTC.hpp:40:10: fatal error: gromacs/fileio/xtcio.h: No such file or directory
 #include "gromacs/fileio/xtcio.h"
          ^~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
src/CMakeFiles/_espressopp.dir/build.make:4646: recipe for target 'src/CMakeFiles/_espressopp.dir/io/DumpXTC.cpp.o' failed
make[2]: *** [src/CMakeFiles/_espressopp.dir/io/DumpXTC.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
In file included from /Downloads/espressopp-1.9.5/src/io/bindings.cpp:32:0:
/Downloads/espressopp-1.9.5/src/io/DumpXTC.hpp:40:10: fatal error: gromacs/fileio/xtcio.h: No such file or directory
 #include "gromacs/fileio/xtcio.h"
          ^~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
src/CMakeFiles/_espressopp.dir/build.make:4694: recipe for target 'src/CMakeFiles/_espressopp.dir/io/bindings.cpp.o' failed
make[2]: *** [src/CMakeFiles/_espressopp.dir/io/bindings.cpp.o] Error 1

When using ap-get everthing wokes fine. I also copied the GROMACS include directory (non standard path) to /usr/include and that also worked. 
But sinsce I would like to run simulations on an HPC cluster (Centos 7.2) this workaround is not possible.

Best Marco

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -83,7 +83,7 @@ include_directories(${FFTW3_INCLUDES})
 
 if(WITH_XTC)
   find_package(GROMACS REQUIRED)
-  include_directories(${GROMACS_INCLUDE_DIRS})
+  include_directories(${GROMACS_INCLUDES})
   add_definitions( -DHAS_GROMACS )
 endif()
 

